package.json

```diff
@@ -15,16 +15,20 @@
     "packages/*"
   ],
   "scripts": {
+    "changelog": "auto-changelog --latest-version $(yarn -s print-version)",
```

Comment:
The use of bash subcommands (`$(...)`) makes this command not compatible with Windows (same as before this change), however this can be resolved by fixing CookPete/auto-changelog#106.